components/operator/internal/state/controller_configuration.go

```diff
@@ -28,15 +33,27 @@ func sFnControllerConfiguration(_ context.Context, r *reconciler, s *systemState
 
 func updateControllerConfigurationStatus(r *reconciler, instance *v1alpha1.DockerRegistry) error {
 	spec := instance.Spec
+	storageField := getStorageField(spec.Storage, instance)
 	fields := fieldsToUpdate{
 		{spec.HealthzLivenessTimeout, &instance.Status.HealthzLivenessTimeout, "Duration of health check", ""},
 		{registry.SecretName, &instance.Status.SecretName, "Name of secret with registry access data", ""},
+		storageField,
 	}
 
 	updateStatusFields(r.k8s, instance, fields)
 	return nil
 }
 
+func getStorageField(storage *v1alpha1.Storage, instance *v1alpha1.DockerRegistry) fieldToUpdate {
```

Comment:
I would suggest considering moving the functionality of storage configuration into a separate state function. I think the state function should describe the functionality of a single and small domain and the `storage` configuration looks like such a small domain. Of course, I don't want to force such change. This is a suggestion only.

Reply:
I don't think this change is necessary, at least for now